_shared/hooks/post_gen_project.py

```diff
@@ -123,7 +123,7 @@ def main():
 
         template_ignore_patterns = [".git/*"]
 
-        {%- if cookiecutter._directory == 'pyramid-app' %}
+        {%- if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
```

Comment:
There's several places in the shared templates where a block of code was previously used by the `pyramid-app` cookiecutter but not the `pypackage` one, and now the blocks must be used by _either_ `pyramid-app` or `pyapp` but not `pypackage`.